src/components/ScholarshipForm.jsx

```diff
@@ -60,6 +63,9 @@ function ScholarshipForm({ scholarship }) {
     },
   });
 
+  let lintIssues = {};
+  if (activeStep === 1) lintIssues = lintReqs(formik.values);
```

Comment:
not necessary but It's probably better than calling this everytime there is a change in the description.